README.md

```diff
@@ -130,3 +131,43 @@ problems?
 You can find the development builds also in the 
 [release section](https://github.com/thsmi/sieve/releases). They are marked as 
 pre-release. They will update to newer releases but not to newer pre-releases.
+
+## Sieve Compatible Mail Servers
+
+Sieve comparability normally depends upon the mail server back end. In case it is 

```

Comment:
There is already a extensive list on http://sieve.info/servers which is maintained. So I would prefer linking to there.

It also feels to me that it is too much text for a starting page. So I would suggest to strip that down to a sentence like this:

"Almost all open-sourced mail servers support sieve. A (incomplete) list can be found at [Sieve.info](http://sieve.info/servers). You won't find any proprietary and locked products (Exchange, Gmail, etc.). But keep in mind even if your server is included in the list the admin may have deactivated it."

Reply:
done